core/src/services/oss/writer.rs

```diff
@@ -41,7 +41,8 @@ pub struct OssWriter {
 }
 
 impl OssWriter {
-    pub fn new(core: Arc<OssCore>, path: &str, op: OpWrite, buffer_size: Option<usize>) -> Self {
+    pub fn new(core: Arc<OssCore>, path: &str, op: OpWrite) -> Self {
+        let buffer_size = core.writer_buffer_size.unwrap_or(8 * 1024 * 1024);
```

Comment:
If we have a default value for `write_min_size`, we should store a `usize` directly.